src/components/UserList.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import UserRecord from './UserRecord'
+import { User } from '../types'
+
+const UserList = ({ users, removeUser }: { users: User[], removeUser: (id: number) => void}) => {
+    return (
+        <div>
+            <h3>User list</h3>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Surname</th>
+                        <th>E-mail</th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {users.map(user => (
+                        <UserRecord
```

Comment:
you need to add unique `key` to each element returned in map

Reply:
The User type has id. It's used to delete a row in a table. Do you mean the key should be added explicity as a parameter?